jobs/dtd-metrics/src/metrics/total-connections.metric.ts

```diff
@@ -0,0 +1,25 @@
+import { AgreementState } from '@interop-be-reports/commons'
+import { MetricFactoryFn } from '../services/metrics-producer.service.js'
+import { getMonthsAgoDate, getVariationPercentage } from '../utils/helpers.utils.js'
+import { z } from 'zod'
+
+export const getTotalConnectionsMetric: MetricFactoryFn<'connessioniTotali'> = async (readModel) => {
+  const agreements = await readModel.agreements
```

Comment:
not blocking this PR, but considering the growing number of agreements it would be more efficient to execute two queries that return just the two totals